apiserver/params/params.go

```diff
@@ -384,6 +384,7 @@ type AddApplicationUnits struct {
 	ApplicationName string                `json:"application"`
 	NumUnits        int                   `json:"num-units"`
 	Placement       []*instance.Placement `json:"placement"`
+	AttachStorage   []string              `json:"attach-storage,omitempty"`
```

Comment:
Since we are adding a new field, strictly speaking the facade version should be bumped. I *think* the pyjuju api lib will be affected by this change otherwise.